src/main/java/uk/gov/hmcts/reform/sendletter/entity/LetterRepository.java

```diff
@@ -14,7 +14,8 @@
     @Lock(LockModeType.PESSIMISTIC_WRITE)
     Stream<Letter> findByState(LetterState state);
 
-    Optional<Letter> findById(UUID id);
 
-    Optional<Letter> findByIdAndService(UUID id, String service);
+    Optional<Letter> findOptionalById(UUID id);
```

Comment:
Is this method used? There are no code updates related with it (other than tests).

Reply:
i saw it coming in another pr